layouts/index.html

```diff
@@ -31,7 +31,7 @@
           See how the tidyverse makes data science faster, easier and more fun with "R for Data Science".
           Read it <a href="http://r4ds.had.co.nz/">online</a>, buy <a href="http://amzn.to/2aHLAQ1">the book</a> or try another <a href="/learn/">resource from the community</a>.
         </div>
-        <a href="http://amzn.to/2aHLAQ1"><img class="bookCover" src="images/cover.png"></a>
+        <a href="http://amzn.to/2aHLAQ1"><img class="bookCover" src="images/cover.png" alt="R for Data Science book cover"></a>
```

Comment:
I think this could be `alt=""` because the image doesn't add anything not already in the text?